docs/docs/user/migration-guide/v2-v3.md

```diff
@@ -40,6 +40,22 @@ All Helm-specific parts of the built-in [`KubernetesApp`](../core-concepts/compo
       ...
 ```
 
+## [Create StreamsBootstrap component & refactor cleanup jobs as individual HelmApp](https://github.com/bakdata/kpops/pull/398)
+
+Previously the default `KafkaApp` component configured the [streams-bootstrap](https://bakdata.github.io/streams-bootstrap/) Helm Charts. Now, this component is no longer tied to Helm (or Kubernetes). Instead, there is a new `StreamsBootstrap` component that configures the Helm Chart repository for the components that use it, e.g. `StreamsApp` and `ProducerApp`. If you are using non-default values for the Helm Chart repository or version, it has to be updated as shown below.
```

Comment:
What is the point of having KafkaApp then?

Reply:
implement the logic related to Kafka, i.e. creation of output topics